### DISCUSSION TITLE
Update of continent (CONT) field

### DISCUSSION THREAD

**Discussion by pa8s** [magicbug/Cloudlog#1935] [General]
Just a question, does Cloudlog update the CONT field when adding/updating a QSO? I have some early entries in my logbook with this field set but I think it was done with CQRLOG, which I've used before switching to Cloudlog. For most of my QSOs this field is empty.

**Comment by AndreasK79** [comment]
As far as I can tell when checking the code, it is imported if CONT is set in the ADIF, but it is not used anywhere else.

**Reply by pa8s** [reply]
I thought something like that. There is already a possibility to update the DXCC of QSOs, couldn't the CONT field be filled in at the same time?

**Reply by phl0** [reply]
It could. See: https://github.com/magicbug/Cloudlog/pull/1969. Please test and report.

**Comment by pa8s** [comment]
Did a quick check and `<my-url>/index.php/update/check_missing_dxcc` resulted in a white empty page with nothing more than `0 updated`, no theme, nothing. 

Checking `<my-url>/index.php/update/check_missing_dxcc/all` resulted in an error page after a few seconds: 
```
An error occurred.

Sorry, the page you are looking for is currently unavailable.
Please try again later.

If you are the system administrator of this resource then you should check the error log for details.

Faithfully yours, nginx.
```
Maybe a timeout issue? My logbook is just shy of 6500 QSOs and server has resources enough. I do have all relevant php timeout values set to 1200 seconds as far as I know. 

Both URLs were accessed via the Check and Re-check buttons on the DXCC Lookup Data menu page.

Will do some more testing and checking tomorrow.

**Reply by phl0** [reply]
Don't think so. The result pages are simply plain text. That is works-as-designed. You did checkout my branch with the patch? Is your DXCC data updated?

**Reply by pa8s** [reply]
Checked on old installation and there a list of updates are shown in plain text, my mistake. My DXCC data is up to date and your patch active.

**Reply by phl0** [reply]
And still no updates? I just pushed a small debug output for the skipped updates. Maybe that reveals the issue.

**Reply by pa8s** [reply]
Check and it works now, deleted col_cont and col_country data of a QSO and with 'Check QSOs missing DXCC data' adds the correct country and continent :D 

'Re-check all ...' still produces timeout error but this might be an local config issue I need to resolve.

**Comment by phl0** [comment]
Made a PR for this: https://github.com/magicbug/Cloudlog/pull/1979. Tests and feedback welcome :)
